reactor-core/src/test/java/reactor/core/publisher/FluxHandleTest.java

```diff
@@ -685,6 +687,18 @@ public void nextAfterErrorFused() {
 		                                                    .hasMessage("Cannot emit after a complete or error"));
 	}
 
+    @Test
+    public void runtimeExceptionFused() {
+        // Check issue that for fuseable and Exception thrown inside handle it did not propagate signal -> hanging flux
+        //  and throws IllegalStateException: Timeout on blocking read
+        Assertions.assertThatThrownBy(() -> {
```

Comment:
you can use the more idiomatic `Assertions.assertThatNullPointerException.isThrownBy` here.
You could then also import the static method rather than `Assertions` as a result.